DEVELOPMENT.md

```diff
@@ -255,6 +255,20 @@ Returns HTTP code 500 if an error occurred getting the extensions
 Returns HTTP code 200 and the given extensions in the given namespace if found, otherwise an empty list is returned
 ```
 
+__Dashboard Properties__
+```
+GET /v1/properties
+Get the properties of the tekton-dashboard which are install namespace, dashboard version and tekton pipelines version
```

Comment:
Nit: 

Gets the properties of the Tekton Dashboard which includes the:

- Installation namespace
- Tekton Dashboard version
- Tekton Pipelines version

Everything else is looking good now 👍

Reply:
Updated now